Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -397,14 +397,40 @@ instance instLinearOrderedCommMonoidWithZero [CommMonoid α] [LinearOrder α] [I
 instance instLinearOrderedCommGroupWithZero [CommGroup α] [LinearOrder α] [IsOrderedMonoid α] :
     LinearOrderedCommGroupWithZero (WithZero α) where
 
-end WithZero
+/-! ### Exponential and logarithm -/
+
+variable {G : Type*} [Preorder G] {a b : G}
+
+local notation "Gₘ" => Multiplicative G
+local notation "Gᵐ⁰" => WithZero Gₘ
```

Comment:
I confess I am lost here: why for the "with zero" you want both `m` and `0` as exponent, whereas for Multiplicative you want `m` as subscript? Also, in GroupWithZero/WithZero, you scoped these (and rightly so, IMHO), but here they're just local notations. Can you explain the rationale?